src/components/AllArticles.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import Header from "./Header";
+import Footer from "./Footer";
+import { Link } from "react-router-dom";
+import { CarouselItem } from "react-bootstrap";
+import { Carousel } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+export default function AllArticles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articleData) => {
+      console.log(articleData, "in function");
+      setArticles(articleData);
+    });
+  }, []);
+
+  return (
+    <section>
+      <Header />
+      <Carousel>
+        {articles.map(({ title, article_img_url, article_id }) => {
+          return (
+            <CarouselItem key={article_id} className="carousel">
```

Comment:
The carousel is making it quite difficult to view and navigate at the moment
Please change it to a list, you could maybe adding a carousel **in addition** to a list of articles if you're set on it later
One big reason for this is later we'll ask you to sort articles by certain properties (like date created, or number of comments) and this won't be possible in a carousel